### PR TITLE
net_node: update seed nodes

### PR DESCRIPTION
**Pull request by selsta**
If someone wants their node added please add a comment under this PR. We especially need Tor / I2P addresses.

### CODE REVIEW

**Comment by d5e005db6f50b85b8f4361cf70a813a5** [comment]
Hello,

riolxz7dd77kwnfhh4troinrdjxcunsinoo2gczqntzlwjgcdv4umdad.onion will be ready once it finishes syncing via Tor, likely 48 hours from now.

Any questions at all, send an email to d5e005db6f50b85b8f4361cf70a813a5@proton.me

**Comment by plowsof** [comment]
this is node3.monerodevs.org:
77.172.183.193 (18089 / 28089 / 38089)
plowsofe6cleftfmk2raiw5h2x66atrik3nja4bfd3zrfa2hdlgworad.onion:18083
ugnlcdciyhghh2zert7c3kl4biwkirc43ke33jiy5slnd3mv2trq.b32.i2p:18080

**Comment by Coffeeplease01** [comment]
Hi,

I'm low-key but almost always around for about two years now. You can use my node if you like:
aclc4e2jhhtr44guufbnwk5bzwhaecinax4yip4wr4tjn27sjsfg6zqd.onion

DM on matrix  for more info if you like.

Regards,
-coffeeplease-

**Comment by selsta** [comment]
@Coffeeplease01 thank you, I added your node.

@d5e005db6f50b85b8f4361cf70a813a5 I have to talk to others if we can add fully anonymous seed nodes, there are small risks with it. Either way, your node will benefit the network.

**Comment by d5e005db6f50b85b8f4361cf70a813a5** [comment]
Okay no problem. Happy to chat more about a cryptocurrency node tor service somewhere other than github too, if you like.

- Either way, your node will benefit the network.

:)

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by nahuhh** [comment]
> ugnlcdciyhghh2zert7c3kl4biwkirc43ke33jiy5slnd3mv2trq.b32.i2p:18080 
> 

should be
`ugnlcdciyhghh2zert7c3kl4biwkirc43ke33jiy5slnd3mv2trq.b32.i2p` no?

File: src/p2p/net_node.inl [hunk 1]
```diff
@@ -857,23 +857,22 @@ namespace nodetool
...
       return {};
     case epee::net_utils::zone::i2p:
       if (m_nettype == cryptonote::MAINNET)
       {
         return {
-          "s3l6ke4ed3df466khuebb4poienoingwof7oxtbo6j4n56sghe3a.b32.i2p:18080",
-          "sel36x6fibfzujwvt4hf5gxolz6kd3jpvbjqg6o3ud2xtionyl2q.b32.i2p:18080",
           "uqj3aphckqtjsitz7kxx5flqpwjlq5ppr3chazfued7xucv3nheq.b32.i2p:18080",
           "vdmnehdjkpkg57nthgnjfuaqgku673r5bpbqg56ix6fyqoywgqrq.b32.i2p:18080",
+          "ugnlcdciyhghh2zert7c3kl4biwkirc43ke33jiy5slnd3mv2trq.b32.i2p:18080",
```
Comment (nahuhh) [inline src/p2p/net_node.inl:875]:
```suggestion
          "uqj3aphckqtjsitz7kxx5flqpwjlq5ppr3chazfued7xucv3nheq.b32.i2p",
          "vdmnehdjkpkg57nthgnjfuaqgku673r5bpbqg56ix6fyqoywgqrq.b32.i2p",
          "ugnlcdciyhghh2zert7c3kl4biwkirc43ke33jiy5slnd3mv2trq.b32.i2p",
```

18080 seems misconfigured.

for onion, were using the port which were binding/broadcasting anon-inbound. Should be the same for i2p. So this should be 0/blank or the provided anon-inbound i2p port (example, mine is 18085).

**Review by plowsof — APPROVED** [review]

**Pull request merged**